components/SearchForm/SearchForm.vue

```diff
@@ -26,6 +31,7 @@ export default {
 
 .search-form {
   display: flex;
+  align-items: center;
```

Comment:
I don't know why, but this made the search form's input field smaller in height. Adding this was correct because the search form should definitely be vertically aligned. Just need to find out why it is affecting the input's height.